src/parser/jobs/rpr/modules/DeathGauge.tsx

```diff
@@ -33,30 +32,31 @@ const VOID_ACTIONS: ActionKey[] = [
 	'LEMURES_SCYTHE',
 ]
 
+const LEMURE_SHROUD_COLOR = Color('#4ec8dc')
+const VOID_SHROUD_COLOR = Color('#b614c4')
```

Comment:
What was wrong with the existing colours?

Reply:
It felt kludgy/fragile to be basing the colors off, in part, other jobs' defined color. They also felt... dull? I guess is the best word to describe it. These were color-picked from a screenshot of the actual gauge UI, similar to how I set up the Astral Fire/Umbral Ice/Feather/Addersting gauge colors.